plugins/ibis/superduper_ibis/data_backend.py

```diff
@@ -331,21 +219,105 @@ def get_table_or_collection(self, identifier):
 
     def disconnect(self):
         """Disconnect the client."""
-
         # TODO: implement me
 
-    def list_tables_or_collections(self):
+    def list_tables(self):
```

Comment:
We need to retain at least one version of the old API; otherwise, upgrading superduper will break the existing application.

Reply:
Which methods are you missing?